can-validate-validatejs.md

```diff
@@ -1,21 +1,71 @@
 @module {Object} can-validate-validatejs
 @parent can-ecosystem
-@group can-validate-validatejs.core 1 can-validate-validatejs Core
 @package ./package.json
 
-@description A plugin for CanJS that wraps any validation library to `can.validate`.
-**Can-Validate doesn't do any validation of its own** but instead provides some abstraction to your library of choice. The chosen library is registered with can-validate using a shim.
+@description Create [can-validate/validator] functions using [validate.js](https://validatejs.org/).
 
-@type {Object}
+@signature makeValidator(constraints)
+
+Returns a validator function that can be used to validate a single value.
+
+@param {Object} constraints An object of definitions used by the Validate.js library to run validations on a value.
+
+@return {Validator} A validator type function that takes a value and validates it against the provided constraints.
+
+@signature makeValidator.many(constraints)
+
+Returns a validator function that can be used to validate many values at once.
+
+@param {Object} constraints A map of objects which contain definitions used by the Validate.js library to run validations on a value.
+
+@return {Validator} A validator type function that takes an object of values and validates every value based on the provided constraints.
+
+
+@body
+
+## Usage
+
+A validator can be created either for a single value or for many values.  More information on how to configure constraints can be found [here](https://validatejs.org/#validators).
+
+### Single value validation
+
+Using the Validate.js library validators and configuration, call `makeValidator` and pass the desired constraints. The resulting function can then be used to validate specific values.
 
 ```javascript
-var Validate = require('can-validate-validatejs');
-var validate = new Validate();
+var validateAge = makeValidator({
```

Comment:
I think it would be helpful to include the `require` statement in both of the examples so it’s clear from where `makeValidator` is coming.